app/controllers/application_controller.rb

```diff
@@ -17,23 +17,25 @@
 
 class ApplicationController < ActionController::Base
     protect_from_forgery
-    before_action :authorize
+    before_action :login_authorization
```

Comment:
I suggest to move the update of access control to another PR, and we can continue there.

Reply:
No issues, will revert